test/functional/query-builder/locking/query-builder-locking.ts

```diff
@@ -99,6 +99,21 @@ describe("query builder > locking", () => {
         }
     })));
 
+    it("should attach dirty read lock statement on query if locking enabled", () => Promise.all(connections.map(async connection => {
+        if (connection.driver instanceof AbstractSqliteDriver || connection.driver instanceof CockroachDriver)
+            return;
+
+        const sql = connection.createQueryBuilder(PostWithVersion, "post")
+            .setLock("dirty_read")
+            .where("post.id = :id", { id: 1 })
+            .getSql();
+
+        if (connection.driver instanceof SqlServerDriver) {
```

Comment:
We limit drivers on wchich we want the test to run in two separate statements. It should be done only in one place.

Reply:
I do not know what the correction would be, any suggestions?

Reply:
What I had in mind is line 103 - we're limiting there some drivers, and here in if condition we choose only one driver after all. We could put this if condition in the beginning and there will be only one place which controls test execution based on driver.

Reply:
Do you mean to put row 103 in a generic way and import it from those test cases?

Reply:
File updated